src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -746,6 +745,10 @@ FixedwingPositionControl::set_control_mode_current(const hrt_abstime &now, bool
 		return; // do not publish the setpoint
 	}
 
+	FW_POSCTRL_MODE last_position_control_mode = _control_mode_current;
```

Comment:
Small nitpick, but this is not actually the mode the vehicle was on the previous time step, but the vehicle is in the current time step. The function `set_control_mode_current` is deciding which mode the vehicle should transition next to

Reply:
renamed. [1d44cb7](https://github.com/PX4/PX4-Autopilot/pull/19869/commits/1d44cb73cb651c0d4401552673b53a0491938335)